extensions/application/test_application.lua

```diff
@@ -49,7 +49,7 @@ function testBasicAttributes()
   return success()
 end
 
-function testActiveAttributes() 
+function testActiveAttributes()
```

Comment:
```suggestion
local function testActiveAttributes()
```

Reply:
I believe it was triggered due to the whitespace change as this was function was not added here. I saw this output and checked to see how common this was:
```
 ~/oss/hammerspoon/extensions $ git grep "^function test" | wc -l
     254
```

I'll gladly open another PR to address this, or even add to this with another commit addressing these instances if that feedback is requested.

Reply:
@landongrindheim - Yes, anytime you change a line of code, @stickler-ci will review it.

Looking at `test_application.lua` more closely, it seems all the functions are global - so this might be a requirement of the way tests work? @cmsj or @asmagill will know more, as I haven't actually played with Hammerspoon's tests much yet.

If we can't make them local, then the solution would be to add `-- luacheck: ignore` at the end of those lines of code, so that @stickler-ci ignores them.

I'll leave @cmsj to decide the best way to tackle.

Reply:
The test functions need to be global, they get called from Objective C automagically based on method names. We could change that so the `test_foo.lua` returns a table of functions I suppose, but I'm not in a hurry to do so ;)